### PR TITLE
TimeTrail Sound Setting

### LINKED ISSUE #97: Make sounds during timetrials optional
Add a field in ts_players to save the setting.

### PR DESCRIPTION
**Pull request by Pigalala**
Adds `/tt sound` which toggles the pb sound. Setting is stored in ts_players : ttSound.
Testing may still be needed.

Closes #97 

Databases will need to run the following statement for this update to work:
``ALTER TABLE `ts_players` ADD `ttSound` VARCHAR(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT 'true' AFTER `boat`;``

### CODE REVIEW

File: src/main/java/me/makkuusen/timing/system/ApiUtilities.java [hunk 1]
```diff
@@ -37,7 +37,8 @@ public class ApiUtilities {
     static TimingSystem plugin;
     private static final String[] months = {"January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"};
     private static final Pattern niceLocation = Pattern.compile("^\\(\\[([A-Za-z0-9_]+)\\]([\\-]{0,1}[0-9]+),[ ]{0,1}([\\-]{0,1}[0-9]+),[ ]{0,1}([\\-]{0,1}[0-9]+)\\)$");
-    private static final List<String> rejectedWords = Arrays.asList("random", "r", "cancel", "c", "help", "verbose");
+    private static final List<String> rejectedWords = Arrays.asList("random", "r", "cancel", "c", "help", "verbose", "sounds");
```
Comment (Makkuusen) [inline src/main/java/me/makkuusen/timing/system/ApiUtilities.java:40]: sounds should only be sound

Comment (Pigalala) [reply src/main/java/me/makkuusen/timing/system/ApiUtilities.java:40]: brah
I'll fix tomorrow

Comment (Pigalala) [reply src/main/java/me/makkuusen/timing/system/ApiUtilities.java]: ok it is fixed now i hope you're happy

**Review by Makkuusen — CHANGES_REQUESTED** [review]
As ttSound is stored as true/false I would prefer if it was done the same way as other booleans like toggleOpen in ts_tracks. I also think it would be cool to rename "ttsound" to "toggleSound".

So for the database table creation add this instead " `toggleSound` tinyint(1) NOT NULL"

And then update the constructor of TPlayer to be data.get("toggleSound"), and make any necessary changes to the setTTSound command as well.

**Comment by Pigalala** [comment]
``ALTER TABLE `ts_players` ADD `toggleSound` tinyint(1) DEFAULT 1 AFTER `boat`;`` i haven't tested this statement i hope it works.
/tt togglesound

**Pull request merged**